forum/qa-plugin/report-reason/q2apro-flag-reasons-layer.php

```diff
@@ -0,0 +1,174 @@
+                    <div class="qa-flag-reasons-wrap">
+                        <h4>
+                            '.qa_lang('q2apro_flagreasons_lang/reason').'
+                        </h4>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="1" checked>
+                            <span>'.q2apro_flag_reasonid_to_readable(1).'</span>
+                        </label>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="2">
+                            <span>'.q2apro_flag_reasonid_to_readable(2).'</span>
+                        </label>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="3">
+                            <span>'.q2apro_flag_reasonid_to_readable(3).'</span>
+                        </label>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="4">
+                            <span>'.q2apro_flag_reasonid_to_readable(4).'</span>
+                        </label>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="5">
+                            <span>'.q2apro_flag_reasonid_to_readable(5).'</span>
+                        </label>
+                        <label>
+                            <input type="radio" name="qa-spam-reason-radio" value="6">
+                            <span>'.q2apro_flag_reasonid_to_readable(6).'</span>
+                        </label>
+                        
+                        <div class="qa-spam-reason-text-wrap">
+                            <p>
+                                '.qa_lang('q2apro_flagreasons_lang/note').'
+                            </p>
+                            <input type="text" name="qa-spam-reason-text" class="qa-spam-reason-text" placeholder="'.qa_lang('q2apro_flagreasons_lang/enter_details').'">
+                        </div>
+                        
+                        <input type="button" class="qa-gray-button qa-go-flag-send-button" value="'.qa_lang('q2apro_flagreasons_lang/send').'">
+                        
+                        <div class="closer">X</div>
+                    </div>
+                </div> 
+            </div>
+            ');
+        }
+        qa_html_theme_base::body_hidden();
+        
+    }
+    
+    public function post_tags($post, $class)
+    {
+        qa_html_theme_base::post_tags($post, $class);
+ini_set('display_errors', 1);
+        if($class === 'qa-q-view') {
+            $postId = $post['raw']['postid'];
+            $flagReasons = q2apro_get_postflags($postId);
+            
+            if(!empty($flagReasons)) {
+                $flagsOut = '
+                    <ul class="qa-flagreason-list">
+                ';
+                
+                foreach ($flagReasons as $f) {
```

Comment:
stosujmy ładne nazewnictwo zmiennych a nie jednoliterkowce

Reply:
No, to akurat pozostałość po starym skrypcie